### PR TITLE
Tab issue

### PR DESCRIPTION
**Pull request by rishabhemr**
This pull request is for the below issues 
#251 #290 #260

Rishabh Software

### CODE REVIEW

**Comment by MatthewVita** [comment]
@rishabhemr Hello. You may want to internationalize the jQuery text. Here's an example of how I accomplished this elsewhere in the codebase: https://github.com/MatthewVita/openemr/blob/5748aabbfc068b8ca20fa0a2cb12974ac02c3a2f/interface/product_registration/product_registration_controller.js#L39

Also, how is this testing for you?

File: sites/default/sqlconf.php [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'openemr';
-$pass	= 'openemr';
-$dbase	= 'openemr';
+$pass	= '123456789';
```
Comment (MatthewVita) [inline sites/default/sqlconf.php]: Oops, you won't want to check in this file.

File: interface/main/tabs/main.php [hunk 2]
```diff
@@ -204,6 +216,27 @@ function isEncounterLocked( encounterId ) {
            $('#attendantData').slideToggle();
             $('#patient_caret').toggleClass('fa-caret-down').toggleClass('fa-caret-up');
         });
+        
+        //Javascript for the tab drag and drop
+        $('.sorting').sortable({
```
Comment (MatthewVita) [inline interface/main/tabs/main.php]: Isn't this the same code as `makeTabsSortable`?

Comment (rishabhemr) [inline interface/main/tabs/main.php]: Hello, @MatthewVita  
yes, makeTabsSortable its a same code for making tab  sortable. But we have made new function in the interface\main\tabs\js\tabs_view_model.js function 

Thanks
Rishabh Software

File: sites/default/sqlconf.php (hunk 1 above)
Comment (rishabhemr) [inline sites/default/sqlconf.php]: Hello @MatthewVita 
Sorry for making check in the configuration file
Let us know what need to do make it effective.

Thanks
Rishabh Software

Comment (MatthewVita) [inline sites/default/sqlconf.php]: Rishabh,

Thanks for replying back.

You're going to want to undo your changes to the file. I would actually favor a "low tech" solution here by copying the following overtop your file and checking it in (thus, reverting your changes): https://raw.githubusercontent.com/openemr/openemr/master/sites/default/sqlconf.php

One tip I have for you is to put `sqlconf.php` on your Desktop and drag it into `sites/default/` when actively developing. Just be careful not to check it in.

Matthew

**Comment by MatthewVita** [comment]
Hi @rishabhemr,

Just so you know, I'd like you to do the following:

- Revert your changes to the sqlconf file (I just put a note on how to do this)
- Add internationalization to "Are You Really Want To Close This Tab?" in interface/main/tabs/templates/tabs_template.php. Also, please reword it to "Are you sure you want to close this tab?".
- Internationalize the jQuery text. Here's an example of how I accomplished this elsewhere in the codebase: https://github.com/MatthewVita/openemr/blob/5748aabbfc068b8ca20fa0a2cb12974ac02c3a2f/interface/product_registration/product_registration_controller.js#L39

Let me know if you need any help and I'll be glad to assist.

Thanks,
Matthew
